include/message.php

```diff
@@ -67,11 +67,8 @@ function send_message($recipient = 0, $body = '', $subject = '', $replyto = '')
 		$fields = array('uid' => local_user(), 'guid' => $conv_guid, 'creator' => $sender_handle,
 			'created' => datetime_convert(), 'updated' => datetime_convert(),
 			'subject' => $subject, 'recips' => $handles);
-		dba::insert('conv', $fields);
-
-		$r = dba::selectFirst('conv', ['id'], ['guid' => $conv_guid, 'uid' => local_user()]);
-		if (DBM::is_result($r)) {
-			$convid = $r['id'];
+		if (dba::insert('conv', $fields)) {
+			$convid = dba::lastInsertId();
```

Comment:
That's much too easy using that function ;-)

Reply:
I know right! Programming should only about pain and suffering!